webui/assets/main.js

```diff
@@ -320,31 +327,35 @@ function render_status(status) {
 
   var header = document.createElement('tr')
   markup = `<th>Host</th>
-  <th>State</th>`
+  <th>ID</th>
+  <th>Coordinator</th>`
   header.innerHTML = markup
   tbody.appendChild(header)
   for(var n=0; n<nodes.length; n++) {
     var row = document.createElement("tr")
-    markup = `<td>${nodes[n]["Host"]}</td>
-    <td>${nodes[n]["State"]}</td>`
+    markup = `<td>${nodes[n]["uri"]["host"]}:${nodes[n]["uri"]["port"]}</td>
+    <td>${nodes[n]["id"]}</td>
+    <td>${nodes[n]["isCoordinator"]}</td>`
```

Comment:
I think there is a cleaner way to indicate the coordinator node. Instead of having "true" and "false" in its own column, "(coordinator)" could be inserted to the right of, or below, the hostname.

Do we not want node state visible here?

Reply:
Good thinking, I'll go ahead and merge this and take that into consideration as I work on the set coordinator/remove node interactions.